client/src/components/post/hooks/usePostActions.js

```diff
@@ -0,0 +1,26 @@
+import { useQuery } from 'react-query';
+import { getDislikes, getLikes } from '../api';
+
+const usePostActions = ({ postId }) => {
+  const { data: { data: { likes } = {} } = {} } = useQuery(
+    [`getLikes-${postId}`, { postId }],
+    getLikes,
+    {
+      staleTime: 10 * 60 * 1000,
+      refetchInterval: 2 * 60 * 1000,
```

Comment:
Why are u refetching every 2 min when the stale time is set on 10 min? Stale time means that the data is fresh for 10min.

Reply:
okay then we make the staleTime also 2 mins